crypto/sha/keccak1600.c

```diff
@@ -11,6 +11,10 @@
 #include <string.h>
 #include <assert.h>
 
+size_t SHA3_absorb(uint64_t A[5][5], const unsigned char *inp, size_t len,
+                   size_t r);
+void SHA3_squeeze(uint64_t A[5][5], unsigned char *out, size_t len, size_t r);
+
```

Comment:
At *this* point it would make my life more complicated. It's still in active development (rather platform support), which means that I regularly have to tailor complier commands, or in other words type them manually. Having to add additional flag would make it only harder...